go/enclave/crypto/crypto.go

```diff
@@ -87,3 +87,21 @@ func decryptWithPrivateKey(ciphertext []byte, priv *ecdsa.PrivateKey) ([]byte, e
 	}
 	return plaintext, nil
 }
+
+// GeneratePublicRandomness - generate 32 bytes of randomness, which will be exposed in the rollup header.
+func GeneratePublicRandomness() []byte {
+	return randomBytes(gethcommon.HashLength)
+}
+
+// PrivateRollupRnd - combine public randomness with private randomness in a way that protects the secret.
+func PrivateRollupRnd(publicRnd []byte, secret []byte) []byte {
+	return crypto.Keccak256Hash(publicRnd, secret).Bytes()
+}
+
+func randomBytes(length int) []byte {
+	byteArr := make([]byte, length)
+	if _, err := rand.Read(byteArr); err != nil {
+		panic(err)
```

Comment:
Do you think panicking is the best option here? I'm not sure, I guess so.

Reply:
good point.